src/backend/tasks_io/datafeeds/datafeed_fms_api.py

```diff
@@ -72,6 +72,8 @@ class DatafeedFMSAPI:
         "dar": "darwin",
         "gal": "galileo",
         "hop": "hopper",
+        "joh": "johnson",
```

Comment:
This also needs to map to the new codes. May need to do something like `EVENT_SHORT_EXCEPTIONS_2023`